### PR TITLE
Fix quick tags in OCP not working on Search/Explore

### LINKED ISSUE #1153: One-Click Postage does not work properly on the "standard" search pages. Also cosmetic issues.
When going to a standard search page or the explore page, One-Click Postage shows a broken, non-working layout (Quick Tags can't be clicked).

The search page in general doesn't look too spiffy, likely because of the multitude of bottom-post icons we add, making each column basically non-stop.

### PR DESCRIPTION
**Pull request by ThePsionic**
Fixes #1153 sorta since 2 issues yadayada.

Ugly fix but does the job, and not too bad in terms of readability.

### CODE REVIEW

File: Extensions/quick_tags.js [hunk 1]
```diff
@@ -67,7 +67,9 @@ XKit.extensions.quick_tags = new Object({
 
 	run: function() {
 		this.running = true;
-		if ($(".post.post_full").length <= 0) {return; }
```
Comment (nightpool) [inline Extensions/quick_tags.js]: this is really huge. Can we do `.post.post_full, .post.post_brick` or something?

File: Extensions/xkit_patches.js [hunk 2]
```diff
@@ -2136,6 +2137,10 @@ XKit.tools.getParameterByName = function(name){
 					m_return.search = true;
 				}
 
+				if (document.location.href.indexOf("tumblr.com/explore/") !== -1) {
```
Comment (nightpool) [inline Extensions/xkit_patches.js]: we should use a body class here like the other pages (likes, dashboard)

Comment (ThePsionic) [inline Extensions/xkit_patches.js]: Both blog and search do not use a body class either, though.

File: Extensions/quick_tags.js (hunk 1 above)
Comment (ThePsionic) [inline Extensions/quick_tags.js]: Changed for your pleasure.

File: Extensions/xkit_patches.js (hunk 2 above)
Comment (nightpool) [inline Extensions/xkit_patches.js]: I would prefer to have a body class over not having a body class, if one exists on the page

Comment (ThePsionic) [inline Extensions/xkit_patches.js]: Changed to `body.discover`

File: Extensions/quick_tags.js [hunk 3]
```diff
@@ -67,7 +67,8 @@ XKit.extensions.quick_tags = new Object({
...
+
+		if ($(".post.post_full").length <= 0 || $(".post.post_brick").length <= 0) { return; }
```
Comment (nightpool) [inline Extensions/quick_tags.js]: Sorry, missed this the first time around. This should be ".post.post_full, .post.post_brick", right? as a selector, rather then OR-ing two selectors together

Comment (ThePsionic) [inline Extensions/quick_tags.js]: TIL. I even had to do a JSFiddle to confirm this actually works lol

**Comment by nightpool** [comment]
@homu r+

**Pull request merged**